src/plugins/home/server/services/tutorials/lib/tutorial_schema.ts

```diff
@@ -156,6 +156,7 @@ export const tutorialSchema = schema.object({
   customStatusCheckName: schema.maybe(schema.string()),
 
   integrationBrowserCategories: schema.maybe(schema.arrayOf(schema.string())),
+  eprPackageOverlap: schema.maybe(schema.string()),
```

Comment:
Could we add a comment here just to make sure there's no confusion about what `eprPackageOverlap` means?